unit-tests/ModelsRelationsTest.php

```diff
@@ -374,6 +374,13 @@ protected function _testIssue938($di)
 		}
 	}
 
+	protected function _testIssue10800()
```

Comment:
yes I realize that, but I was unsure when that might happen or if it would break any tests so I added to both unit-tests and tests, in the future I will make changes only to tests.